src/features/wallet/SideBarMenu.tsx

```diff
@@ -210,31 +208,15 @@ function TransferSummary({
         {STATUSES_WITH_ICON.includes(status) ? (
           <Image src={getIconByTransferStatus(status)} width={25} height={25} alt="" />
         ) : (
-          <SmallSpinner className="-ml-1 mr-3" />
+          <>
```

Comment:
Why the fragment?

Reply:
derp, forgot to remove it after testing something